app/metrics.yaml

```
@@ -0,0 +1,80 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+---
+$schema: moz://mozilla.org/schemas/glean/metrics/2-0-0
+
+no_lint:
+  - CATEGORY_GENERIC
+
+metrics:
+  default_browser:
```

We could, for example, have a "browser" category and group some metrics under it :)

```suggestion
browser:
  is_default:
```